### PR TITLE
add plenty more core metrics to ship from telegraf to datadog

### PR DESCRIPTION
**Pull request by oryband**
- **important** new environment variables that need to be passed: see docker-compose file for additions
- in addition, cleaned up the telegraf app configuration files, custom metrics scripts, and Dockerfiles for better readability
- renamed old metrics to new one to not override existing ones
with alerts on them on datadog
- also added complete telegraf setup to local docker-compose for testing and debugging when necessary

### CODE REVIEW

File: images/dockerfiles/stellar-core-telegraf/Dockerfile [hunk 1]
```diff
@@ -1,10 +1,18 @@
 FROM telegraf:1.9.4-alpine
```
Comment (fuzzyami) [inline images/dockerfiles/stellar-core-telegraf/Dockerfile]: the latest is 1.9.5

File: images/dockerfiles/stellar-core-telegraf/input-exec/get-app-state-synced.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```
Comment (fuzzyami) [inline images/dockerfiles/stellar-core-telegraf/input-exec/get-app-state-synced.sh]: the first section is totally over-engineered and requires too much bash. bash is terrible.

Comment (oryband) [reply images/dockerfiles/stellar-core-telegraf/input-exec/get-app-state-synced.sh]: i removed the switch. please comment on the exact line so github could mark them as "outdated" once it's fixed

File: images/dockerfiles/stellar-core-telegraf/Dockerfile (hunk 1 above)
Comment (oryband) [reply images/dockerfiles/stellar-core-telegraf/Dockerfile]: done

**Pull request merged**